spec/content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/update_status_spec.rb

```diff
@@ -65,6 +75,27 @@
 
       msg = "Server [#{miq_server.name}] Service [#{svc_service.name}] Provision Step [] Status [fred] "
       expect(svc_model_request.reload.message).to eq(msg)
+      expect(svc_service).not_to receive(:on_error)
+    end
+  end
+
+  context "on_error provisioning" do
+    let(:root_object) do
```

Comment:
You can move `let(:root_object)` out of the context block, and assign `'service_action' => service_action`. Then within each context block you only need to have `let(:service_action)  { 'Provision/Retirement'}`